src/main/kotlin/no/nav/syfo/infrastructure/database/repository/VurderingRepository.kt

```diff
@@ -45,6 +45,19 @@ class VurderingRepository(private val database: DatabaseInterface) : IVurderingR
             }
         }
 
+    override fun createVurdering(vurdering: Vurdering, pdf: ByteArray?) {
```

Comment:
Synes det er god stil at alt som lager noe returnerer det de lager slik at det kan brukes videre om man trenger 😁 Skal vi legge på vurderingen som bli laget her? 🤌

Reply:
Vi kan godt det! Foreløpig har vi ikke bruk for responsen noe sted, ettersom man sender inn vurderingen man uansett vil få ut igjen. I så fall må vi legge på et ekstra kall når vi mapper om fra `pVurdering` til `Vurdering` for å hente tilhørende `Varsel`.